CONTRIBUTING.rst

```diff
@@ -57,11 +57,11 @@ You need Python 3.8+.
    $ python -m venv .venv
    $ source .venv/bin/activate
 
-Install in editable mode:
+Install in editable mode including development dependencies:
 
 .. code:: console
 
-   $ pip install -e .
+   $ pip install -e .[dev]
```

Comment:
Just because dev also installs all frameworks and it is probably too much in most cases, I think would make more sense to:

```suggestion
   $ pip install -e .[tests]
```

Although I should update the framework tests to be skipped when the framework is missing